### ISSUE TITLE
[Bug] Unable to connect to covered wagon as either player, player leading horse or player riding horse

### ISSUE THREAD

**Issue by KaelaniRevyruun** [capnkirok/animaniamod#526]
### Expected Behavior

- Connect to wagon as player, or connect horse to wagon when riding draft horse, or have horse connect when wagon is right clicked with leashed horse

### Actual Behavior

- No action performed (Potentially wagon hitbox not registering interaction)

### Steps to Reproduce

-Place covered wagon
- Attempt to connect to wagon

### Version of Minecraft, version of Animania, Single Player or Server

Minecraft version: 1.12.2 (Tekxit 3.14 Beta modpack version 0.13.2)
Animania version: 2.0.3.28
Server: ApertureGaming minecraft server

**Comment by Tschipp** [comment]
how exactly are you attemping to connect the wagon?

**Comment by KaelaniRevyruun** [comment]
I'm attempting to connect the wagon by right clicking on the beam that leads to the two handles that stick out, and going all the way back to the main body of the wagon itself.

**Comment by Tschipp** [comment]
so right clicking the wagon does nothing?

**Comment by KaelaniRevyruun** [comment]
Yep, tried while not riding a horse, tried while riding a horse, tried right clicking with a leash attached to the horse, did nothing.

**Comment by benjamineddy** [comment]
Also experiencing this issue. Modpack Tekxit 3.14 version 1.0.5 and animania version 2.0.3.28

**Comment by DubStepMad** [comment]
An confirm the issue happens on the modpack Tekxit 3.14
